app-connector-client/server/routes/events.js

```diff
@@ -48,11 +49,12 @@ async function createEventsFromConfig(localKyma, eventsConfig, registeredApis) {
 
         } catch (error) {
             LOGGER.error("Registration of Event API '%s' failed: %s", event.name, JSON.stringify(error))
+            throw error
         }
     }
 }
 
-function createEvent(localKyma, eventMetadata, event) {
+async function createEvent(localKyma, eventMetadata, event) {

```

Comment:
what is the reason for async? No await command is used

Reply:
removed